layouts/partials/footer.html

```diff
@@ -20,10 +20,13 @@
         {{ with .Site.Params.copyright }}
         <div class="row">
           <div class="col-12">
-            <small class="text-white">&copy; {{ now.Year}} {{ .}} {{ T "footer_all_rights_reserved" }}</small>
+            <small class="text-white">&copy; {{ now.Year}} The cert-manager Authors</small>
           </div>
           <div class="col-12">
-            <small class="text-white">cert-manager was originally created by <a href="https://jetstack.io"><span id="jetstack-footer">{{ with resources.Get "icons/jetstack.svg" }}{{ ( . | minify).Content | safeHTML }}{{ end }}</span></a></small>
+            <small class="text-white">&copy; {{ now.Year}} The Linux Foundation. All rights reserved. The Linux Foundation has registered trademarks and uses trademarks.</small>
+          </div>
+          <div class="col-12">
+            <small class="text-white">The Linux Foundation has registered trademarks and uses trademarks. For a list of trademarks of The Linux Foundation, please see our <a class="text-white" href="https://www.linuxfoundation.org/trademark-usage">Trademark Usage page</a></small>
```

Comment:
Is `The Linux Foundation has registered trademarks and uses trademarks. For a list of trademarks ...` correct? So many 'trademark' 😆

Reply:
it's their core business ;)